src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -94,6 +94,8 @@ private void assertGpOutboundQueueMessages(SoftAssertions softly, Message messag
                 new Customization("identifier.value", IGNORE),
                 new Customization("entry[*].fullUrl", IGNORE),
                 new Customization("entry[*].resource.subject.reference", IGNORE),
+                new Customization("entry[*].resource.performer.reference", IGNORE),
+                new Customization("entry[*].resource.requester.agent.reference", IGNORE),
```

Comment:
Are there any references we would like to test on? If not, could we make this a blanket rule to ignore any reference?

Reply:
It would be nice to verify the reference equals the ID of the target resource, but that sounds complicated. Failing that, it's a nice idea to see if something like `entry[*].resource.*.reference` would work. Might have to check the relevant documentation

Reply:
@andrewwa-kainos it will. Fixed in a5fa4c30c0599727e597931467a8da16dd52af28
Thanks!